ext/reflection/php_reflection.c

```diff
@@ -1664,6 +1664,59 @@ ZEND_METHOD(ReflectionFunctionAbstract, getClosureScopeClass)
 }
 /* }}} */
 
+/* {{{ Returns an associative array containing the closures lexical scope variables */
+ZEND_METHOD(ReflectionFunctionAbstract, getClosureUsedVariables)
+{
+	reflection_object *intern;
+	const zend_function *closure_func;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+	GET_REFLECTION_OBJECT();
+
+	array_init(return_value);
+	if (!Z_ISUNDEF(intern->obj)) {
+		closure_func = zend_get_closure_method_def(Z_OBJ(intern->obj));
+		if (closure_func == NULL ||
+			closure_func->type != ZEND_USER_FUNCTION ||
+			closure_func->op_array.static_variables == NULL) {
+			return;
+		}
+
+		const zend_op_array *ops = &closure_func->op_array;
+
+		HashTable *static_variables =
+			ZEND_MAP_PTR_GET(ops->static_variables_ptr);
+
+		if (!static_variables) {
+			return;
+		}
+		
+		zend_op *opline = ops->opcodes + ops->num_args;
```

Comment:
Might want to double-check that this works under `-E` ... I think you may run into trouble with the extra ops that inserts.

Reply:
Good catch, it looks like enough to include EXT_STMT in continuation condition ...

Reply:
Actually, EXT_STMT is not inserted between BIND_STATICS immediately following prologue ... so it was fine the way t was ... I extended the test, although don't know how to run it with extended info automatically ...